components/headers/GatesHeader.vue

```diff
@@ -50,6 +50,14 @@ export default class extends Vue {
 <style lang="scss" scoped>
 @import '~/assets/scss/mixins.scss';
 
+.extra-container {
+  margin-right: 1rem;
+}
+
+.copy-container:not(:only-child) {
```

Comment:
I'm not very happy with this solution since in the case we add another page and it has two childs, this will happen as well. I understand why it's made like this since it's more difficult to add a new class depending on the page. I'm approving it by now. We can find a better solution in the future. 😬

Reply:
Actually I found a way of doing it.
Adding this code: 
```
:class="{ 'whatever-class-name': $nuxt.$route.name === 'education' }
```
to [line 13](https://github.com/qiskit-community/community.qiskit.org/blob/084b20174820cf924d87accda2c79545575d99f4/components/headers/GatesHeader.vue#L13) will allow you to add the needed class depending on the route.